sdl_android/src/main/java/com/smartdevicelink/managers/SdlManager.java

```diff
@@ -427,15 +459,16 @@ public SdlManager build() {
 				sdlManager.hmiLanguage = Language.EN_US;
 			}
 
-			sdlManager.state = BaseSubManager.SETTING_UP;
+			sdlManager.transitionToState(BaseSubManager.SETTING_UP);
 
 			return sdlManager;
 		}
 	}
 
 	private void checkSdlManagerState(){
-		if (state != BaseSubManager.READY){
-			throw new IllegalStateException("SdlManager is not ready for use, be sure to initialize with start() method, implement callback, and use SubManagers in the SdlManager's callback");
+		if (state != BaseSubManager.READY && state != BaseSubManager.LIMITED){
```

Comment:
The state should be accessed through the `getState` method here since there is synchronization around the getting and setting.